src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -47,6 +50,13 @@
     private final AtomicReference<ConnectionManager> connectionManager = new AtomicReference<>();
     private final AtomicReference<SubscriberMonitor> consumerMonitor = new AtomicReference<>();
 
+    private final MetricArbiter livenessArbiter = CommonMetrics.getLIVENESS_ARBITER();
+    private final MetricArbiter readinessArbiter = CommonMetrics.getREADINESS_ARBITER();
+    private MetricArbiter.MetricMonitor livenessMonitor;
+    private MetricArbiter.MetricMonitor readinessMonitor;
+
+    private final ScheduledExecutorService executor = new ScheduledThreadPoolExecutor(2);
```

Comment:
This field should take from init method, because we can create much subscribers and each will use 2 thread for this field. Please take count of thread from message router settings.